build.sbt

```diff
@@ -1,24 +1,24 @@
 name := """tdr-transfer-frontend"""
 organization := "tna"
+maintainer := "tdr@nationalarchives.gov.uk"
```

Comment:
Is this compulsory? If you haven't already, can you talk to David about what email address we should use?

Reply:
It's not mandatory but it was the last warning the build was giving. I've asked David and he's given me the proper email for this.

Reply:
+1 for fixing compiler warnings. Thanks!